### PR TITLE
ENH: Regenerate using latest SlicerCustomAppTemplate

### PR DESCRIPTION
**Pull request by jadh4v**
Regenerate the project using the latest changes in
SlicerCustomAppTemplate. Template version used:
KitwareMedical/SlicerCustomAppTemplate@1fb5751

Core Slicer version bumped from Slicer/Slicer@b22191e
to Slicer/Slicer@acc6180

Addresses issue: https://github.com/KitwareMedical/vpaw/issues/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+## Maintainers
+
+* [Contributing](CONTRIBUTING.md)
+* [Building](BUILD.md)
 
 
-References
+![vpaw by Kitware, Inc.](Applications/vpawApp/Resources/Images/LogoFull.png?raw=true)
 
+## References
```
Comment (jcfr) [inline README.md:29]: Add a license section:

```
## License

This software is licensed under the terms of the [Apache License Version 2.0](LICENSE).
```

Comment (jcfr) [inline README.md:29]:
```suggestion
## How to cite
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
 1) [QUAMMEN et al, 2017, The Virtual Pediatric Airways Workbench ](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5588666/)
```
Comment (jcfr) [inline README.md:30]:
```suggestion
Quammen, C. W., Taylor, R. M., 2nd, Krajcevski, P., Mitran, S., Enquobahrie, A., Superfine, R., Davis, B., Davis, S., & Zdanski, C. (2016). The Virtual Pediatric Airways Workbench. Studies in health technology and informatics, 220, 295–300.
See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5588666/
```

**Comment by jcfr** [comment]
Thanks for working on this :pray: 

Few additional comments:
- Is the plan to target multiple platform or only windows ?
- Consider including a brief History section. For example, 

```
### History

This initial version of this project was developed between 20XX and 20YY in the context of ..., the project as well as the team involved are respectively described and acknowledged in the [referenced](#how-to-cite) publication.
```

**Comment by jcfr** [comment]
Last, which description should be used here:

image

**Comment by jadh4v** [comment]
Based on the existing readme, it appears to only support windows. We can keep it that way until we build and test for linux and mac in a separate PR?

File: BUILD.md [hunk 3]
```diff
@@ -4,9 +4,12 @@ Build and Package vpaw
 This document summarizes how to build and package vpaw on Windows. Instructions for Linux and macOS are similar.
 For more details, see [3D Slicer Developer Wiki](http://wiki.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers)
 
-Prerequisites 
--------------
+vpaw is a custom Slicer application. Reading the [3D Slicer Developer Documentation](https://slicer.readthedocs.io/en/latest/developer_guide/index.html) may help answer additional questions.
```
Comment (jcfr) [inline BUILD.md:7]: Consider updating this document based of https://github.com/KitwareMedical/SlicerCustomAppTemplate/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/BUILD.md

File: README.md [hunk 4]
```diff
@@ -1,8 +1,35 @@
...
+_List key features_
+
+
+## Table of content
+
+- [vpaw by Kitware, Inc.](#vpaw-by-kitware-inc)
+  - [Features](#features)
+  - [Table of content](#table-of-content)
+  - [Maintainers](#maintainers)
+  - [References](#references)
```
[the remark this answers is not in the document]
Comment (jadh4v) [reply README.md]: You comment is on an outdated commit. Can you refresh and check again?

File: README.md [hunk 5]
```diff
@@ -1,8 +1,45 @@
...
 
-A Virtual Pediatric Airways Workbench (VPAW) is a patient-centered surgical planning software system targeted to pediatric patients with airway obstruction. VPAW is enables a  full surgical planning pipeline, including importing DICOM images, segmenting the airway, interactive 3D editing of airway geometries to express potential surgical treatment planning options, and creating input files for offline geometric analysis and computationalfluid dynamics simulations for evaluation of surgical outcomes.  This repostiory contains a new generation of VPAW that is developed using [3D Slicer](https://www.slicer.org/).
+_This project is in active development and may change from version to version without notice,_
```
Comment (jcfr) [inline README.md:5]:
```suggestion
_This project is in active development and may change from version to version without notice._
```

File: README.md [hunk 6]
```diff
@@ -1,8 +1,45 @@
...
 
+## Features
 
+a) Importing DICOM images \
+b) Segmenting the airway \
+c) Interactive 3D editing of airway geometries to express potiential surgical treatment planning options \
+d) Creating input files for offline geometric analysis and CFD simulations
```
Comment (jcfr) [inline README.md:12]:
```suggestion

The project is being actively developer and our plan is to implement the following functionalities to enable users to express potential surgical treatment planning options for pathology like XX, YYY.

A typical workflow will consist in the following steps:

* Import of DICOM images
* Segmentation of airway
* Interactive 3D editing of airway geometries leveraging integration of XXX device through OpenHaptics interface.
* Export of segmented airways for offline geometric analysis and CFD simulations
```

Comment (jcfr) [reply README.md:12]: @jadh4v Review the description for workflow for accuracy and update accordingly :pray:

Comment (jcfr) [reply README.md:12]: > Import of DICOM images

Is there a specific set of modalities that we plan to support first (CT, MRI, ... ?)

Comment (jcfr) [reply README.md:12]: > Segmentation of airway

Which approach will be available ? Full set of effect available in Slicer segmentation editor ? Or a specific subset ? Or a novel method specific to this project ?

Comment (jadh4v) [reply README.md:12]: > > Import of DICOM images
> 
> Is there a specific set of modalities that we plan to support first (CT, MRI, ... ?)

The reference paper itself only refers to CT scans. MRI is usually more expensive and time consuming as compared to CT. I suspect that CT scans are probably preferred for this treatment. In the ReadMe, we can say:
`* Import of DICOM images (initially we will support only CT data)`

Regarding the segmentation methods included, I think this is something of a future decision and will require input from collaborators on whether we are going to modify the previously established/published approach. For now we could go with what the published paper says. It applies the method described by _Hong, et al._ in _section 2.1_ of https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4769591/

Comment (jadh4v) [reply README.md:12]: @andinet do you want to pitch in on these points?

File: BUILD.md (hunk 3 above)
Comment (jadh4v) [reply BUILD.md:7]: It is generated from the file you mention, but I have retained a few bullet points from the previous version, do you want me to remove them? bullet points listed below:
```
* Microsoft Windows 7 or above recommended

* Supported Microsoft Visual Studio versions:
    * Visual Studio 2015
    * Visual Studio 2017

* [CMake](http://cmake.org/cmake/resources/software.html), version 3.11 or above

* Qt, version 5.10 or above

* [Git](http://git-scm.com/downloads)

```

**Review by jcfr — APPROVED** [review]

**Pull request merged**